scripts/common.js

```diff
@@ -598,3 +598,16 @@ export const isDevHost = () => {
   const devHosts = ['localhost', 'hlx.page', 'hlx.live', 'aem.page', 'aem.live'];
   return devHosts.some((url) => window.location.host.includes(url));
 };
+
+/**
+ * Function that recieves a timestamp in seconds and returns a date
+ * in its locale's format. Defaults to US format date (MM/DD/YYYY)
+ * @param {string} timestamp The date in seconds as a string
+ * @param {object} options The date options for a specific format
+*/
+export const getDateFromTimestamp = (timestamp, options = undefined) => {
```

Comment:
`options = undefined` is redundant 
if you don't pass the param it will be `undefined` by default

Reply:
also true